app/graphql/types/query_type.rb

```diff
@@ -118,11 +119,13 @@ def waste_location_type(id:)
       Waste::LocationType.find(id)
     end
 
-    def news_items_data_providers(category_id: nil)
-      return DataProvider.joins(:news_items).order(:name).uniq if category_id.blank?
+    def news_items_data_providers(category_id: nil, category_ids: nil)
+      if category_id.blank? && category_ids.blank?
+        return DataProvider.joins(:news_items).order(:name).uniq
+      end
 
       NewsItem
-        .by_category(category_id)
+        .by_category(category_id || category_ids)
```

Comment:
die Methode :by_category kann sowohl eine id als auf eine Liste an IDs aufnehmen?

Reply:
yes: `where(categories: { id: category_id })` which works for `where(categories: { id: 1 })` as well as for `where(categories: { id: [1, 2, 3] })`